src/leap/mail/imap/account.py

```diff
@@ -49,6 +49,16 @@ def _debugProfiling(result, cmdname, start):
 # Soledad IMAP Account
 #######################################
 
+class IIMAPAccount(Interface):
+    """
+    The abstraction describing an IMAPAccount.
+    """
```

Comment:
I can get rid of this; I thought at some point that I needed a specific interface for the avatar.